### ISSUE TITLE
Resolve Old Hungarian controversy

### ISSUE THREAD

**Issue by davelab6** [googlefonts/noto-fonts#2065]
There is some controversy about the way Old Hungarian has been implemented. This issue will track the Noto project's progress on this topic.

**Comment by tamasbartos** [comment]
@davelab6 
The Unicode standard does not prescribe mandatory added ligatures, just those, which are in the Old Hungarian chart.
[Old Hungarian chart](https://unicode.org/charts/PDF/U10C80.pdf#page=2)
[Old Hungarian description](https://www.unicode.org/versions/Unicode13.0.0/UnicodeStandard-13.0.pdf#page=377)
Android devices doesn't require Old Hungarian font with additional ligatures. Ligatures are useful only for word processors.
iPhone doesn't use additional ligatures.
The source code of Old Hungarian font without additional ligatures already uploaded, but has rendering-command problems.

**Comment by tamasbartos** [comment]
@davelab6 There are ligatures in NotoSansOldHungarian font, which are useful, but have appereance problems. I will upload tomorrow screenshots.
There are ligatures,

**Comment by tamasbartos** [comment]
@davelab6 @marekjez86 THERE IS A MUST TO REMOVE ALL LEFT TO RIGHT LIGATURES, BECAUSE THERE AREN'T ANY PROGRAM, EDITOR, TERMINAL WHICH USE THESE LIGATURES!

**Comment by tamasbartos** [comment]
@davelab6 There is the first problematical ligature:
![1st problematical ligature](https://user-images.githubusercontent.com/75836867/116204562-2f19dd80-a73d-11eb-9340-05529bbb1841.jpg)
Resolving this problem:
Redefine as "daa",also known as "dá" (uni10CC7_10CC1= U+10CC7 ZWJ U+10CC1) and/or redraw with mirrored "a" (U+10CC0) letterpart, wider than "d" letterpart.

**Comment by davelab6** [comment]
@tamasbartos thanks for posting this. However, as I said before, we will hire someone independent of you and @rovasinfo to investigate this topic. I expect it will be later this year. Until then, this topic is paused.

**Comment by dscorbett** [comment]
In [L2/21-073](https://www.unicode.org/L2/L2021/21073-script-adhoc-rept.pdf#page=20), the Script Ad Hoc group explicitly declines to offer definitive guidance regarding the existence or proper encoding of diacritics in Old Hungarian.

**Comment by twardoch** [comment]
I don't quite follow the logic that if there is no program that can use some glyphs, then the glyphs must be removed from the official distribution.

Anyone is free to subset the font, remove any glyphs they want and publish that modified font.

If these ligatures cannot be used (supposedly), then nobody will ever see them except if they look into the font. So perhaps these ligatures aren’t so useful, but is that reason enough that they “must” be removed? I fail to see that.

And yet I somehow doubt that there is _no program_ that can access the ligatures. XeTeX? LuaLaTeX? InDesign? Chrome? No?

**Comment by tamasbartos** [comment]
> In [L2/21-073](https://www.unicode.org/L2/L2021/21073-script-adhoc-rept.pdf#page=20), the Script Ad Hoc group explicitly declines to offer definitive guidance regarding the existence or proper encoding of diacritics in Old Hungarian.

@dscorbett  That isn't clear for me, what you want to wrote about.

**Comment by tamasbartos** [comment]
@davelab6 I understand, that is it paused, but I give another informations about ligatures.
It seems, that ligatures aren't developed by persons, who know Hungarian language.
For example there are wrong formed ligatures, which aren't useful:
Ligature "ck": 
!["ck" ligaturel](https://user-images.githubusercontent.com/75836867/116775573-ee370700-aa63-11eb-86c9-8a9d890c6190.jpg)
Ligature "np":
!["np" ligature](https://user-images.githubusercontent.com/75836867/116775616-36eec000-aa64-11eb-9755-28cb219e603a.jpg)
 
There are syllable like useful ligatures, for example "ba", "be", "bi", "bo", but aren't implemented for example "baa" (bá), "bee" (bé), "bii" (bí), "boo" (bó), "boe" (bö), "boee" (bő), "bu", "buu" (bú), "bue" (bü) and "buee" (bű).
!["ba" "be" "bi" "bo"](https://user-images.githubusercontent.com/75836867/116775734-d3b15d80-aa64-11eb-9424-7499b5bee591.jpg)
Here is the full list of vowels and consonants, which need for useful syllable like ligatures:
![for syllables](https://user-images.githubusercontent.com/75836867/116775842-96010480-aa65-11eb-8eda-4463a4afddc7.jpg)

**Comment by tamasbartos** [comment]
> There is some controversy about the way Old Hungarian has been implemented. This issue will track the Noto project's progress on this topic.



> @tamasbartos thanks for posting this. However, as I said before, we will hire someone independent of you and @rovasinfo to investigate this topic. I expect it will be later this year. Until then, this topic is paused.

@davelab6  May I resolve UI variant's rendering problems, or UI version is paused,too?
I wait for your answer for this comment only.

**Comment by tamasbartos** [comment]
> > In [L2/21-073](https://www.unicode.org/L2/L2021/21073-script-adhoc-rept.pdf#page=20), the Script Ad Hoc group explicitly declines to offer definitive guidance regarding the existence or proper encoding of diacritics in Old Hungarian.
> 
> @dscorbett That isn't clear for me, what you want to wrote about.



> In [L2/21-073](https://www.unicode.org/L2/L2021/21073-script-adhoc-rept.pdf#page=20), the Script Ad Hoc group explicitly declines to offer definitive guidance regarding the existence or proper encoding of diacritics in Old Hungarian.

@dscorbett My English is not so good. Could you write it with another words?

**Comment by dscorbett** [comment]
I asked Unicode for advice about diacritics in Old Hungarian. In [L2/21-073](https://www.unicode.org/L2/L2021/21073-script-adhoc-rept.pdf#page=20), Unicode’s Script Ad Hoc group responded. They think that the “duplicating mark” should be represented by U+0304. However, they did not say anything about any other diacritics. In fact, they said it’s not Unicode’s responsibility to say anything about Old Hungarian diacritics. The point of my comment, therefore, was that we are not going to get much help from Unicode in resolving this issue.

**Comment by tamasbartos** [comment]
> I asked Unicode for advice about diacritics in Old Hungarian. In [L2/21-073](https://www.unicode.org/L2/L2021/21073-script-adhoc-rept.pdf#page=20), Unicode’s Script Ad Hoc group responded. They think that the “duplicating mark” should be represented by U+0304. However, they did not say anything about any other diacritics. In fact, they said it’s not Unicode’s responsibility to say anything about Old Hungarian diacritics. The point of my comment, therefore, was that we are not going to get much help from Unicode in resolving this issue.

What is the conclusion?

**Comment by tamasbartos** [comment]
> I asked Unicode for advice about diacritics in Old Hungarian. In [L2/21-073](https://www.unicode.org/L2/L2021/21073-script-adhoc-rept.pdf#page=20), Unicode’s Script Ad Hoc group responded. They think that the “duplicating mark” should be represented by U+0304. However, they did not say anything about any other diacritics. In fact, they said it’s not Unicode’s responsibility to say anything about Old Hungarian diacritics. The point of my comment, therefore, was that we are not going to get much help from Unicode in resolving this issue.

@davelab6 @dscorbett 
About Old Hungarian spelling:(In the document is used expression "spell")
In Old Hungarian writing, accents were used only for a short, transitional period. From the end of 19th century to the beginning of 20th century.
They experimented with Old Hungarian writing. In this period appeared Ráduly's text with triangle formed "o" , too.
The standard is based on the alphabet of Sándor Forrai (Hungarian Lifetime Achievement Award winner teacher) This alphabet does not use accents or duplication marks, because that not need.
The historic Old Hungarian texts didn't use accents at all.
Standard forms are used by everyone with minor variations, such as some modern letters from Rovásinfo.

**Comment by tamasbartos** [comment]
> @tamasbartos thanks for posting this. However, as I said before, we will hire someone independent of you and @rovasinfo to investigate this topic. I expect it will be later this year. Until then, this topic is paused.

@davelab6 I hope, you wrote about "Unicode" when wrote "independent" word.

**Comment by tamasbartos** [comment]
@davelab6! The @RovasFoundation is not an independent commenter, the user @rovasinfo (Tamás Rumi) is the leader of Rovas Foundation!

**Comment by tamasbartos** [comment]
@davelab6 I sincerely ask you ask Unicode about the contradictory details of Old Hungarian script.

I repeat my opinion, that addtional ligatures do not required for phone devices. That make imcompatibility.

The Noto-fonts project has no task of standardization.

I hope, Noto-fonts project remove all additional ligatures from variant of Old Hungarian font for phone and tablet devices.

I hope, Noto-fonts project add all syllable like ligatures to Old Hungarian font variant for editors and word processors or remove all ligatures. With another word: do not make a halfwork!

I hope, Noto-fonts project remove all left-to-right ligatures from Old Hungarian font, because these ligatures just increase and/or duplicate the size of font, there are not any programs, editors, word processors, which use it.

**Comment by tamasbartos** [comment]
@davelab6 There is an article about Bonyhai-moga alphabet. On the page 7, table 8 made by  Elek Benkő, archaelogist, member of Magyar Tudományos Akadémia (Hungarian scientific Academy)

Who is the amateur? Dr. Gábor Hosszú engineer or Elek Benkő?

On the page 7, picture 7/2 shows, how manipulated the text. 7/1 is a part from the text.

[VAN-E X, Y, W A SIMÉNFALVI  ROVÁS  BEJEGYZÉSEKBEN.pdf](https://github.com/googlefonts/noto-fonts/files/6531568/VAN-E.X.Y.W.A.SIMENFALVI.ROVAS.BEJEGYZESEKBEN.pdf)

**Comment by tamasbartos** [comment]
@RovasFoundation 
[12014-n4183-hungarian.pdf](https://github.com/googlefonts/noto-fonts/files/6533527/12014-n4183-hungarian.pdf)

That was your submission, wasn't it? Then wat's your problem with the form of "close e" and "h". This submission is professional? Didn't forget anything? Where are the "Nikolsburg oe" and "Rudimenta oe", for example? That  naming convention come from the original finds, as you professionals know well!

**Comment by rovasinfo** [comment]
> @davelab6 There is an article about Bonyhai-moga alphabet. On the page 7, table 8 made by Elek Benkő, archaelogist, member of Magyar Tudományos Akadémia (Hungarian scientific Academy)
> 
> Who is the amateur? Dr. Gábor Hosszú engineer or Elek Benkő?
> 
> On the page 7, picture 7/2 shows, how manipulated the text. 7/1 is a part from the text.
> 
> [VAN-E X, Y, W A SIMÉNFALVI ROVÁS BEJEGYZÉSEKBEN.pdf](https://github.com/googlefonts/noto-fonts/files/6531568/VAN-E.X.Y.W.A.SIMENFALVI.ROVAS.BEJEGYZESEKBEN.pdf)

Please do not lie, the pdf you referred here is written by an amateur, who is not member of the Hungarian Academy of Science, even not member of any university, namely Klara Friedrich.
On the other hand I explained in details about the importance of the letters existing in the current Hungarian alphabet (Q, Y, W, X, DZ, DZS), you may forgot about the closed issues of this script. As @davelab6 asked you kinfly many times, do not start these debates here again, because they are hiring somebody, hopefully not from Unicode, but somebody who knows the history of the Hungarian language and can take distinctions between the Rovas script (living, contemporary script) and the Old Hungarian (latin based script, extinct).

**Comment by tamasbartos** [comment]
> > @davelab6 There is an article about Bonyhai-moga alphabet. On the page 7, table 8 made by Elek Benkő, archaelogist, member of Magyar Tudományos Akadémia (Hungarian scientific Academy)
> > Who is the amateur? Dr. Gábor Hosszú engineer or Elek Benkő?
> > On the page 7, picture 7/2 shows, how manipulated the text. 7/1 is a part from the text.
> > [VAN-E X, Y, W A SIMÉNFALVI ROVÁS BEJEGYZÉSEKBEN.pdf](https://github.com/googlefonts/noto-fonts/files/6531568/VAN-E.X.Y.W.A.SIMENFALVI.ROVAS.BEJEGYZESEKBEN.pdf)
> 
> Please do not lie, the pdf you referred here is written by an amateur, who is not member of the Hungarian Academy of Science, even not member of any university, namely Klara Friedrich.

@davelab6 The article really written by Klára Friedrich. Please concentrate table number 8, which is Elek Benkő's table. He is the member of the Hungarian scientific Academy.
@rovasinfo Please read first the comment carefully, after that write about who the lier.
You refered the "Bonyha-moga"' alphabet, as that contains 600 years old q!


> On the other hand I explained in details about the importance of the letters existing in the current Hungarian alphabet (Q, Y, W, X, DZ, DZS), you may forgot about the closed issues of this script. As @davelab6 asked you kinfly many times, do not start these debates here again, because they are hiring somebody, hopefully not from Unicode, but somebody who knows the history of the Hungarian language and can take distinctions between the Rovas script (living, contemporary script) and the Old Hungarian (latin based script, extinct).

@davelab6 @rovasinfo This comment does not change the fact, that Old Hungarian "Q","W","Y", "X", "DZ", "DZS" letters aren't part of the Unicode Old Hungarian standard.
@rovasinfo Please read on the project's homepage in the FAQ about  Klingon alphabet!

**Comment by tamasbartos** [comment]
@rovasinfo Are you willing compromise? I think not. You hadn't accepted the requested letters by you as ligatures. Those ligatures are not a parts of the Unicode Old Hungarian script standard. You hadn't accepted the forms of letters "h" and "close e", as in your submission had been developed, too.

**Comment by tamasbartos** [comment]
@rovasinfo Sorry, I correct myself, before you wrote about me, that I a lier...
Your hadn't accepted the forms of letter "h" and "close e" as in that subbmission, which prefered by Rovas Foundation, had been developed. Submission, which is a "professional" by Rovas Foundation.

**Comment by tamasbartos** [comment]
> As for Szekely-Hungarian Rovas (Old Hungarian for the time being), indeed it would be time to ask for opinion of acknowledged professionals, scientific researchers instead of activists and enthusiastic amateurs. Recent years, academics at the ELTE (Eötvös Loránd University of Sciences), BMGE (Budapest University of Technical and Economic Sciences) and before everything else the government institute of MKI (Institute for Hungarian Studies - with 2 relevant Research Centres: of the Classical Philology and of the Linguistic Planning) are dealing with this script the highest level, organising conferences and publishing articles, books. Unfortunately, during the standardisation process of the last 10 years, these professional voices were "ignored and excluded" ... for some reasons. As a conclusion, the recent Unicode standard does not fit the needs of neither the user base nor the scientific experts, technology developers. The digital use of the script is paralysed or limited to hack, tinker the standard. No wonder, during the development of Noto Font the controversies (errors, missing characters, wrong naming, etc) do impede the procedure as well. Giving the fact, that this script is a living one and the user base is growing, unless the basic issues are not solved - at least technically -, any incomplete or defective font will be just another subject of tinkering or abolition. However, if the goal to create something useful, the community can count on us.

@davelab6 The MKI founded 1st january in 2019.
Trust was withdrawn in 2012 from Gábor Hosszú.

**Comment by tamasbartos** [comment]
> As for Szekely-Hungarian Rovas (Old Hungarian for the time being), indeed it would be time to ask for opinion of acknowledged professionals, scientific researchers instead of activists and enthusiastic amateurs. Recent years, academics at the ELTE (Eötvös Loránd University of Sciences), BMGE (Budapest University of Technical and Economic Sciences) and before everything else the government institute of MKI (Institute for Hungarian Studies - with 2 relevant Research Centres: of the Classical Philology and of the Linguistic Planning) are dealing with this script the highest level, organising conferences and publishing articles, books. Unfortunately, during the standardisation process of the last 10 years, these professional voices were "ignored and excluded" ... for some reasons. As a conclusion, the recent Unicode standard does not fit the needs of neither the user base nor the scientific experts, technology developers. The digital use of the script is paralysed or limited to hack, tinker the standard. No wonder, during the development of Noto Font the controversies (errors, missing characters, wrong naming, etc) do impede the procedure as well. Giving the fact, that this script is a living one and the user base is growing, unless the basic issues are not solved - at least technically -, any incomplete or defective font will be just another subject of tinkering or abolition. However, if the goal to create something useful, the community can count on us.

@davelab6  @rovasinfo  Standardization's last 10 years? Gábor Hosszú undertake the leading of standardization 2008. Trust was withdrawn in 2012, because he just delayed work, smuggled Khazar letters, etc. Then Michael Everson undertake the leading of standardization. It lasted 6 months.

**Comment by tamasbartos** [comment]
@davelab6 In first 4 years Michael Everson tried to work with Gábor Hosszú, but he can not. Many submissions was rejected, due to  professional errors, too. I don't think, Michael Everson is "amateur". @rovasinfo had had problem with him, that Everson is not Hungarian.

**Comment by tamasbartos** [comment]
@davelab6 Did you ask feedback from Unicode?

**Comment by tamasbartos** [comment]
> No; I won't approach Unicode about this. I'll find an expert who is fully independent, and, to your point about Mr Everson, ideally I will find someone who is indeed Hungarian. But it will be slow. Thanks for your patience.

@davelab6 
It is not a good choice for resolve problems, waiting an "professional developer". As you know now,  @rovasinfo  try to manipulated the debute with a newer user @RovasFoundation as "profesdional" .
There were conferences, but those were organized by "amateurs".
The true is that, everybody "amateur" in this theme, members of Rovas Foundation, too.
I believe, that only one way to resolve this problem to ask Unicode.

**Comment by tamasbartos** [comment]
@davelab6 Please, ask Unicode! That is the only independent organization or person that is truly independent in this theme.

**Comment by tamasbartos** [comment]
> No; I won't approach Unicode about this. I'll find an expert who is fully independent, and, to your point about Mr Everson, ideally I will find someone who is indeed Hungarian. But it will be slow. Thanks for your patience.

@davelab6 I am not sure, I understand, what kind of question do you want answered.

**Comment by tamasbartos** [comment]
@davelab6 Here is the Unicode specification of Old Hungarian script:
https://www.unicode.org/versions/Unicode13.0.0/UnicodeStandard-13.0.pdf#page=377
or:
http://www.unicode.org/versions/Unicode13.0.0/ch08.pdf#page=16

Here is the Unicode chart of Old Hungarian script:
https://unicode.org/charts/PDF/U10C80.pdf#page=2

The Old Hungarian standard did not changed since Unicode version 8.0, since it was introduced.
Any other letters might to be implemented as ligatures.

Letter "w" is not a real ligature, It is activated by "Old Hungarian v" + ZWJ + "Old Hungarian v", which, if a device or software do not implements it, appears two "v". If we choose triple "Old Hungarian v" activator, it will be appears triple "v" on devices, if  do not implements this capability, etc.

I tried to prepare the ligature problems and implement missing ligatures, which have logical and appearance problems.
I don’t know who asked for ligatures at all.

The same baseline of small and capital Old Hungarian letters useful for underlined texts, and teaching for kids.  
(I have got a book from @rovasinfo, and I don't like their centered  aligned letters)
 
You will not find an official body or person in Hungary who is independent and familiar with ligatures. 

@davelab6  May I ask you, what kind of questions would you like answered?

**Comment by tamasbartos** [comment]
@davelab6 I wrote a wrong link about Old Hungarian part of Unicode standard. The good one is:
https://www.unicode.org/versions/Unicode13.0.0/ch08.pdf#page=16
I wrote before with http protocoll instead of https

**Comment by tamasbartos** [comment]
@davelab6 If you really want independent peoples about Old Hungarian standard script, ask Michael Everson or Szabolcs Szelp!

**Comment by viktorkiwikovacs** [comment]
@davelab6 
Must not override Unicode codepoints, because that makes compatibility problems.
There are the facts about Old Hungarian script:
The Unicode standard Old Hungarian part is here:
https://www.unicode.org/versions/Unicode13.0.0/ch08#page=16
The Old Hungarian characters' chart with codepoints is here:
https://unicode.org/charts/PDF/U10C80.pdf#page=2
There aren't codepoints of Old Hungarian letters "q", "w","y","x","dz" and"dzs".

The only way to resolving this problem that these letters define as ligatures with ZWJ. Must not to use automatic ligatures (without ZWJ) because of that, it produce that kind of word  like "tyúxar".
Correct form of this word is "tyúkszar" (The "sz" lettercombination in Latin based Hungarian spelling sounds like English "s") tyúk->hen, szar-> shit, tyúkszar-> hen's shit. I am sorry abou ecample.

Changig letterforms of characters "close e" and "h" requested by @rovasinfo, have not legitimate, because in submission 
https://www.unicode.org/L2/L2012/12014-n4183-hungarian.pdf#page=28
by Dr. Gábor Hosszú already defined their forms as in final standard appeared. @rovasinfo had prefered this document before.

Must not to change form of "rudimenta oe" , because it formed as the one of the main first find appear, the "rudimenta" is name of the find.
The same baseline of small and capital symbols is practical, when the text is underlined. Childrens can read better this kind of variation.

The ligatures during the history were used first, when on the wooden stick has not enough space to write that the writer can.
That kind of phylosophy was born in modernized ligature.

The really practical ligatures are syllables and "q","w","y","x", "dz" and "dzs" ligatures. The others, which are developed in NotoSansOldHungarian-Regular fonts aren't relevant.

The Old Hungarian codeblock of Unicode standard provides using bi-di algorithm. Right to left writing.
The left to right form of Old Hungarian letters used only for lazy programmers, when they don't code bi-di algorithm.
Left to right ligatures never works, I never met programs, which is use left to right ligatures. It just increase the font's size.

**Comment by viktorkiwikovacs** [comment]
@davelab6 I wrote a wrong link in.https://github.com/googlefonts/noto-fonts/issues/2065?_pjax=%23js-repo-pjax-container#issuecomment-873661319
The definition of Old Hungarian dcript is:
https://www.unicode.org/versions/Unicode13.0.0/ch08.pdf#page=16

**Comment by viktorkiwikovacs** [comment]
@davelab6 May I ask you, drop me a comment, on what way will be Noto project's  Old Hungarian script to be developed?

**Comment by viktorkiwikovacs** [comment]
@davelab6 I think, I wrote informations correctly about Old Hungarian script and this situation in
https://github.com/googlefonts/noto-fonts/issues/2065#issuecomment-873661319
and
https://github.com/googlefonts/noto-fonts/issues/2065#issuecomment-873663501
May I ask you to unlock the development?

**Comment by viktorkiwikovacs** [comment]
@rovasinfo Tamás, do you happy now? The development of Old Hungarian font in Noto-fonts project is freezed now. 
I remember, what you wrote me about András Tisza. You wrote "félművelt" about him.
He understood, how ligatures work.
You don't know what "reserved point" means in the Unicode standard.  That is not the same as "freely used." 
What is that ferry tail about hundreds of years old "x" and "q" Rovas ( Old Hungarian) letters?
You know well, ligature "ks" (sounds in English "ksh" ) doesn't match "x". I talk about "bonyha-moga" letters. What was written in Hungarian language with the letter "q" hundreds of years ago? The theme "bonyha moga" is very suitable for misleading people who do not understand Hungarian language. 
When I found this project, I asked you to submit ligature forms because I saw that the ligature database is not complete.
You did not answer, you asked if I was okay.
You wrote me before that you had written an article about wrong formed "close e" and "h". Did you consult Gábor Hosszú? He is your colleauge, when we talk about Old Hungarian script,isn't he?
I talk about his submission:
https://www.unicode.org/L2/L2012/12014-n4183-hungarian.pdf

**Comment by viktorkiwikovacs** [comment]
@rovasinfo @RovasFoundation I would be happy if you did not distribute a code page on "rovas.info" that uses non-standard characters for "reserved" code points.  This kind of behavior just gives us a reason why neither Microsoft nor Google should take us seriously.

**Comment by viktorkiwikovacs** [comment]
@rovasinfo @RovasFoundation If you really want nonstandard form of "Old Hungarian rudimenta oe", "Old Hungarian close e", "Old Hungarian h" and nonstandard letterlike symbols, like "q", "w", "y", "x", "dz", "dzs", why don't you use https://oldhungarian.eu/  page's font implementation? It required a keyboard layout, which have ZWJ key (u+200d) only.
@rovasinfo I still don't understand, why did you use in your books "Old Hungarian h" on that kind of form, which differs from the last submission of Gábor Hosszú. I don't understand, why do you talk about modernising of Old Hungarian script, when you talk about that, Khazar leters was the requirement of Carpatian basin's "rovás"  Unicode block, which submission was discarsed. Who speaks in Khazar language in the Carpatian basin?

**Comment by rovasinfo** [comment]
@viktorkiwikovacs > why don't you use https://oldhungarian.eu/ page's font implementation?
As the standardization process was biased, nonprofessional and mislead, the result is definitely useless, thus not in practical use - so simple. Therefore, until a new, comprehensive standard - that enables ALL users in ALL fields the seamless use of contemporary Rovas - is developed, other solutions has to be used, like our proposal that is already the major practical standard. As a conclusion, in all technical development processes - like Notofont - the same situation has to be solved.

**Comment by rovasinfo** [comment]
@viktorkiwikovacs > still don't understand, why did you use in your books "Old Hungarian h" on that kind of form, which differs from the last submission of Gábor Hosszú.
Do not joke, have you heard about typography? The Unicode standard is not about typography, therefore there are unlimited freedom for typographers, like latin fonts.

On the other hand, I mentioned earlier, if you use "khazar" term in this project, it is a form of antisemitim. If you do not know the history of the scripts belong to the Rovas script family, please avoid to use this term. (Who speaks in Khazar language in the Carpatian basin?)

**Comment by rovasinfo** [comment]
I write again: the rovas alphabet MUST be compatible with current extended latin based Hungarian alphabet, including digraphs (dz, dzs), and ligatures of w, q, y as well. Using ZWJ technology is not part of the solution, it is a problem. This is why we use the reserved empty codepoints. 
As the Unicode code page declares, this script is intended to use in modern form, it supports the casing, what was not used before.

**Comment by RovasFoundation** [comment]
> @davelab6 In first 4 years Michael Everson tried to work with Gábor Hosszú, but he can not. Many submissions was rejected, due to professional errors, too. I don't think, Michael Everson is "amateur". @rovasinfo had had problem with him, that Everson is not Hungarian.

You really do have a lot of time to write long and vicious sermonings without any ground and facts but with full of malice and defamation. This is a professional, technical forum - so please stop it. Instead of it, try to learn, read and if you have real questions you are more than welcome to ask them. Best wishes.

**Comment by RovasFoundation** [comment]
> I'll find an expert who is fully independent, and, to your point about Mr Everson, ideally I will find someone who is indeed Hungarian.

I can just support this initiative. Mr Everson has practically zero knowledge about the Hungarian language, its history, its writing system and about the Rovas - as none of them is his profession. Furthermore, he proved his incapacity to lead, moderate the standardisation process and to achieve acceptable compromise as well. The practical failure of the Unicode standardisation - which will go on as Hungarian authorities and experts already proposed to Unicode - is mainly due to his attitude. There are proper Hungarian institutions, experts to deal with for this issue.

**Comment by RovasFoundation** [comment]
@viktorkiwikovacs 
> @rovasinfo @RovasFoundation I would be happy if you did not distribute a code page on "rovas.info" that uses non-standard characters for "reserved" code points. This kind of behavior just gives us a reason why neither Microsoft nor Google should take us seriously.

The prime definition of a standard is that it satisfies its users. If not, it will be not used. Is that clear enough? The Unicode standard is erroneous, incomplete - thus useless in its recent form. Therefore, it is not used by many users - this is so-so simple. As a solution, alternative character tables with still "non-standard" letters are in practical use which will result the necessary changing in the actual standard until it will be acceptable and usable. That is how the endless standardisation process works :) ... and if Microsoft or Google do not take seriously the recent stage of standard and chaos around it - this is the best thing can happen to Rovas. Please note: the proper job has to be done first!

**Comment by RovasFoundation** [comment]
@viktorkiwikovacs 
> I don't understand, why do you talk about modernising of Old Hungarian script, when you talk about that, Khazar leters was the requirement of Carpatian basin's "rovás" Unicode block, which submission was discarsed. Who speaks in Khazar language in the Carpatian basin?

Please take your time to study carefully the history of the standardisation of Rovas - especially as a script family. Yes, this is a serious profession - it will cost you time. Than, you will understand how it works and probably you will be able to ask questions instead of stating nonprofessional opinions and malicious propaganda. As for the Khazar (Khazarization) issue as well, please take your time to make your homework properly. 

The same time, I give you a warning: in the Rovas standardisation process, the misuse of the Khazarian issue - so called "Khazarisation attempt" and other calumniations - was precisely documented as extreme politically driven actions by Neonazi and Antisemitic actors. Thus, next occasion you actively refer to these means, you will be reported officially violating the rules of Github.

**Comment by RovasFoundation** [comment]
@viktorkiwikovacs 
> @rovasinfo Tamás, do you happy now? The development of Old Hungarian font in Noto-fonts project is freezed now.

And finally today, although I was not addressed by this question, I can give you a feed back as well: Yes, the right decision by Noto-fonts was to freeze the project. This way there will be time for the decision makers to carefully investigate the situation and find the proper experts to clear the professional questions. This way, Noto-fonts will be able to avoid the same failure that Unicode was not. We really hope, that Noto-fonts decision makers fully understand that if a defective, erroneous and incomplete font would be accepted, it would be not the solution but a new problem. And as a conclusion, the vast majority of the contemporary users would (could) not use Noto-fonts ...

**Comment by RovasFoundation** [comment]
@viktorkiwikovacs 
> @RovasFoundation @rovasinfo The use of ligature definitions is the only one way to display "q", "w", "y", "x", "dz", "dzs" letters. No one of the large IT companies will push letters inside to the reserved points of Unicode. You know, not only LO can use ligature definitions, it can use more internet browsers, except MS Edge.

Once more ... and again: contemporary Rovas do use all the required letter for lot of reasons, like compatibility to the Latin-base Hungarian orthography, Latin-Rovas interscription of digital contents, usage of data bases, etc. Thus, not ligatures are the solution but the individual code point. There is no space to bargain about it.

The former administrators of the Unicode standardisation process were not capable to respect these facts. ... not yet. For the time being, the recent standard is still useless and the issues have to be resolved soon, next month, next year, in 5 years... whenever. Until it happens, the contemporary users do not use the Unicode standard, nor anything (e.g. erroneous Noto-font) based on it.

And do not worry about "big IT companies", they do not care about this issue anyway. The main point are the needs of the real user base (not noisy activists) and the innovations by the developers: they have already found alternative solutions to most of the recent problems and definitely will solve all the challenges with OR without Unicode, other players.

That's what is called "practical standard" ... try to make your peace with it :)

> Please read https://google.com/get/noto/help/faq what is written about Klingon alphabet.
> @davelab6 As you can read, @RovasFoundation and @rovasinfo don't care what's in the standard, not interested in how to solve the problem.

**Comment by viktorkiwikovacs** [comment]
> @viktorkiwikovacs
> 
> > I don't understand, why do you talk about modernising of Old Hungarian script, when you talk about that, Khazar leters was the requirement of Carpatian basin's "rovás" Unicode block, which submission was discarsed. Who speaks in Khazar language in the Carpatian basin?
> 
> Please take your time to study carefully the history of the standardisation of Rovas - especially as a script family. Yes, this is a serious profession - it will cost you time. Than, you will understand how it works and probably you will be able to ask questions instead of stating nonprofessional opinions and malicious propaganda. As for the Khazar (Khazarization) issue as well, please take your time to make your homework properly.
> 
> The same time, I give you a warning: in the Rovas standardisation process, the misuse of the Khazarian issue - so called "Khazarisation attempt" and other calumniations - was precisely documented as extreme politically driven actions by Neonazi and Antisemitic actors. Thus, next occasion you actively refer to these means, you will be reported officially violating the rules of Github.

YOU WRITE ABOUT NEONAZISM? from whom were the Old Hungarian place name tables "Jobbik" ordered?

**Comment by viktorkiwikovacs** [comment]
> @viktorkiwikovacs
> 
> > @RovasFoundation @rovasinfo The use of ligature definitions is the only one way to display "q", "w", "y", "x", "dz", "dzs" letters. No one of the large IT companies will push letters inside to the reserved points of Unicode. You know, not only LO can use ligature definitions, it can use more internet browsers, except MS Edge.
> 
> Once more ... and again: contemporary Rovas do use all the required letters for a lot of reasons, like compatibility to the Latin-base Hungarian orthography, Latin-Rovas interscription of digital contents, usage of data bases, etc. Thus, not ligatures are the solution but the individual code point. There is no space to bargain about it.
> 
> The former administrators of the Unicode standardisation process were not capable to respect these facts. ... not yet. For the time being, the recent standard is still useless and the issues have to be resolved soon, next month, next year, in 5 years... whenever. Until it happens, the contemporary users do not use the Unicode standard, nor anything (e.g. erroneous Noto-font) based on it.
> 
> And do not worry about "big IT companies", they do not care about this issue anyway. 

I don't think. The noto-fonts project was started by Google for developing fonts their opsystems, like Android and Chrome-OS.

>The main points are the needs of the real user base (not noisy activists, ideologists) and the innovations of the developers: they have already found alternative solutions to most of the recent problems and definitely will solve all the challenges with OR without Unicode, other players.
> 
> That's what is called "practical standard" ... try to make your peace with it :)
> 
> > Please read https://google.com/get/noto/help/faq what is written about Klingon alphabet.
> > @davelab6 As you can read, @RovasFoundation and @rovasinfo don't care what's in the standard, not interested in how to solve the problem.

**Comment by RovasFoundation** [comment]
@viktorkiwikovacs 
> YOU WRITE ABOUT NEONAZISM? from whom were the Old Hungarian place name tables "Jobbik" ordered?
Yes, I warned you about the well documented facts during the Rovas standardisation process where de facto extremist actions, Antisemitic campaign were lead by supporters of Everson. Part of this means are the fuelled by the "Khazar" issue.

Once again boy: watch you words!

**Comment by viktorkiwikovacs** [comment]
@rovasinfo Tamás! 1st submit an application to Unicode, wait for it to be accepted.  Then discuss the necessary code overrides with the developers of Google's "Noto-fonts" project.

**Comment by viktorkiwikovacs** [comment]
@rovasinfo Until your standard change is accepted, you can still apply the ligature definitions.

**Comment by twardoch** [comment]
1. Please be respectful! 

2. As we’ve stated many times, **Noto** follows **Unicode**, but allows for additional glyphs like ligatures. But the core character set is what’s in Unicode. 

3. The Noto development team is not the right place to raise problems with the Unicode standardization. I realize there may be controversies, but it’s simply pointless to debate this **here**, because of the simple principle: Noto follows Unicode. If Unicode changes, the Noto team will consider adjusting the fonts. 

4. In your discussion here, please **stay on topic**. As the Noto font developers, we cannot really take any position in a political or ideological conflict. I realize scripts are often connected to group identities, and discussion about history need to happen. But this is not the right place for these discussions — because, except you, nobody is able to distill these discussions! The topic here is whether the Noto Old Hungarian fonts are correctly implementing the Unicode Standard. Also, if some ligatures are on by default but a community agrees that they should be optional (discretionary). Also discussions about a specific **shape** (form) of particular characters are acceptable. 

5. What’s **not on topic** are discussions about whether the Unicode Standard is correct, whether the encoded character set of Noto Old Hungarian fonts is correct (if that character set follows Unicode). Also, we don’t expect to **remove** glyphs — but we can implement some glyph forms as stylistic alternates (to accommodate different group conventions), and we can implement some ligatures as discretionary (optional) so they’re not on by default. Accusing each other of Antisemitism and Nazism is also not on topic and not acceptable — this does not help your efforts at all. At some point, if you guys start shouting and throwing insults, everybody will just stop listening. 

Thank you.

**Comment by davelab6** [comment]
Thanks Adam. As I said in May, we will address this, but it will be slow. Nothing changed on our side since then. I'll unlock this issue when we are ready to move forward. I'll also close and lock any other issues opened about Old Hungarian until then.
